.github/workflows/Makefile

```
@@ -0,0 +1,16 @@
+	@chmod -f +w $@ || true
+	m4 $< > $@
+# Squeeze multiple blank lines into one.
+	cat -s $@ > $@.tmp
+# Remove leading blank lines.
+	sed '/[^[:blank:]]/,$$!d' $@.tmp > $@
+# Remove trailing blank lines.
+	sed -e :a -e '/^\s*$$/{$$d;N;ba' -e '}' $@ > $@.tmp
+	mv -f $@.tmp $@
+	@chmod -w $@
+
+clean:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)